packages/frontend/src/components/campaign-creation-form/rewards/RewardsPicker.vue

```diff
@@ -50,14 +53,21 @@ const { data: rawBalances, loading: loadingBalances } = useReadContracts(
     })),
 );
 
-const tokensWithBalance = computed(() => {
-    if (!whitelistedTokens.value) return [];
+watch(loadingBalances, () => {
```

Comment:
Don't we have `vue-use` as a dep? We could use their composable to achieve a debouncing effect.

Reply:
Sounds good, addressed in 8f89c27c22b497e2d0984b1941f45a3639c80e5e.